app/helpers/admin/orders_helper.rb

```diff
@@ -0,0 +1,14 @@
+module Admin
+  module OrdersHelper
+    def order_adjustments(order)
+      order.adjustments.eligible.select do |adjustment|
+        type = adjustment.originator_type
+
+        is_shipping_method_fee = (type == 'Spree::ShippingMethod')
```

Comment:
I think it's important we are strict with naming so for me there's a big difference between shipping_method_fee, shipping_method_adjustment and shipping_fee. Fee's (just like hubs) dont exist in ofn code, we only have adjustments not fees. So I'd call this variable is_shipping_method_adjustment.

Reply:
I agree, I added another commit (and then made it obsolete :wink:)